src/averageAges.js

```diff
@@ -37,7 +38,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(female => withChildren
```

Comment:
It's recommended to use a ternary operator instead of `&&` to make the code more readable. Replace with `female.sex === 'f' && (withChildren ? people.some(child => child.mother === female.name) : true)`